lib/parser.js

```diff
@@ -498,6 +498,24 @@ Parser.prototype = {
       && (this.lookahead(i + 1).space || 'indent' == this.lookahead(i + 2).type))
       return false;
 
+    // Nested property $p.color property
+    if (
+      fromProperty
+    ) {
+      let j = i + 1, node;
+      while((node = this.lookahead(j)) && node.type !== 'newline' && node.type !== 'eos') {
+        if (
+          'ident' == node.type &&
+          node.val && node.val.name &&node.val.name.indexOf('$') === 0 &&
+          '.' == this.lookahead(j + 1).type &&
```

Comment:
do you actually need this part? You know it starts with `$` and it's an ident.

Reply:
Yes, unfortunately in language there is no difference between
```
a = {
    padding: 10px
}
  button,
   a.padding,
   input [type = submit],
   input [type = button]
     padding a.padding;
```
And this fix only works if variables always start with a dollar sign.
```
$a = {
    padding: 10px
}
  button,
   a.padding,
   input [type = submit],
   input [type = button]
     padding $a.padding;
```
if you know how do it better, I will be very grateful

Reply:
I fixed it